config/locales/forms/contact_postcode_forms/en.yml

```diff
@@ -2,10 +2,9 @@ en:
   contact_postcode_forms:
     new:
       title: Contact address postcode
-      heading: What's the address of the person we should contact?
+      heading: What's the contact address?
       detail_subheading: Why do you need to write to me?
-      detail_paragraph_1: We’ll only write to you in relation to your waste carrier account. For example, if you order copy cards of your registration.
-      detail_paragraph_2: Don’t worry, we won’t send junk mail to this address.
+      detail_paragraph: We’ll only write to you about your waste carrier account, for example if you order copy cards.
```

Comment:
I know that's what it said before as well, but just occurred to me – should we be saying "registration cards" instead of "copy cards"?